app/src/main/java/org/mifos/mobile/ui/activities/PassCodeActivity.kt

```diff
@@ -15,15 +30,42 @@ import org.mifos.mobile.utils.Constants
 import org.mifos.mobile.utils.MaterialDialog
 import org.mifos.mobile.utils.Toaster
 
+@RequiresApi(Build.VERSION_CODES.M)
```

Comment:
Will cause crash on api lower than M. Pull out the biometric thing seperate like [this](https://gist.github.com/rahul-gill/cb2c5f2fe564b07bb3f982d42dc05ea5), check if device has biometric capabilities and then based on that, show the biometric prompt

Reply:
@rahul-gill on it